src/palette/internal/palettecreator.cpp

```diff
@@ -1530,65 +1530,35 @@ PalettePtr PaletteCreator::newTextPalette(bool defaultPalette)
-    sp->appendElement(overdrive, QT_TRANSLATE_NOOP("palette", "Overdrive"))->setElementTranslated(true);
-
-    auto harmonics = makeElement<PlayTechAnnotation>(gpaletteScore);
-    harmonics->setXmlText(QT_TRANSLATE_NOOP("palette", "harmonics"));
-    harmonics->setTechniqueType(PlayingTechniqueType::Harmonics);
-    sp->appendElement(harmonics, QT_TRANSLATE_NOOP("palette", "Harmonics"))->setElementTranslated(true);
-
-    auto jazzTone = makeElement<PlayTechAnnotation>(gpaletteScore);
-    jazzTone->setXmlText(QT_TRANSLATE_NOOP("palette", "jazz tone"));
-    jazzTone->setTechniqueType(PlayingTechniqueType::JazzTone);
-    sp->appendElement(jazzTone, QT_TRANSLATE_NOOP("palette", "Jazz tone"))->setElementTranslated(true);
-
-    auto normal = makeElement<PlayTechAnnotation>(gpaletteScore);
-    normal->setXmlText(QT_TRANSLATE_NOOP("palette", "normal"));
-    normal->setTechniqueType(PlayingTechniqueType::Natural);
-    sp->appendElement(normal, QT_TRANSLATE_NOOP("palette", "Normal"))->setElementTranslated(true);
+    struct PlayTechAnnotationInfo {
+        muse::TranslatableString xmlText;
+        PlayingTechniqueType playTechType;
+        bool expressionType = false;
+    };
+
+    static const std::vector<PlayTechAnnotationInfo> playTechAnnotations = {
+        { muse::TranslatableString("palette", "legato"),    PlayingTechniqueType::Legato },
+        { muse::TranslatableString("palette", "pizz."),     PlayingTechniqueType::Pizzicato },
+        { muse::TranslatableString("palette", "arco"),      PlayingTechniqueType::Natural },
+        { muse::TranslatableString("palette", "tremolo"),   PlayingTechniqueType::Tremolo, true },
+        { muse::TranslatableString("palette", "mute"),      PlayingTechniqueType::Mute }, //: For brass and plucked string instruments: staff text that prescribes to use mute while playing, see https://en.wikipedia.org/wiki/Mute_(music)
+        { muse::TranslatableString("palette", "open"),      PlayingTechniqueType::Open }, //: For brass and plucked string instruments: staff text that prescribes to play without mute, see https://en.wikipedia.org/wiki/Mute_(music)
+        { muse::TranslatableString("palette", "distort"),   PlayingTechniqueType::Distortion },
+        { muse::TranslatableString("palette", "overdrive"), PlayingTechniqueType::Overdrive },
+        { muse::TranslatableString("palette", "harmonics"), PlayingTechniqueType::Harmonics },
+        { muse::TranslatableString("palette", "jazz tone"), PlayingTechniqueType::JazzTone },
+        { muse::TranslatableString("palette", "normal"),    PlayingTechniqueType::Natural },
+    };
+
+    for (PlayTechAnnotationInfo playTechAnnotation : playTechAnnotations) {
+        auto pta = makeElement<PlayTechAnnotation>(gpaletteScore);
+        if (playTechAnnotation.expressionType) {
+            pta->setTextStyleType(TextStyleType::EXPRESSION);
+        }
+        pta->setXmlText(playTechAnnotation.xmlText.translated());
```

Comment:
This breaks re-translatability, i.e. when switching to another language. 
The _untranslated_ text should be specified here (which is what QT_TRANSLATE_NOOP does: it just returns its second argument). By calling `setElementTranslated(true)`, the element's text will be translated later on.
(one could argue that that method should have been called <code>setElementTranslat<I>able</I></code>...)